Task8/Task8/Program.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+
+namespace Task8
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Task 1");
+            Console.WriteLine("Введите целое число от 0 до 500");
+            var a = int.Parse(Console.ReadLine());
+            int sumCu = 0;
+
+           for (int i = a; i < 500; i++)
+            {
+                sumCu += i * i * i;
+            }
+            Console.WriteLine($"Сумма кубов чисел от {a} до 500 равна {sumCu}");
+            Console.WriteLine();
+
+            ///Console.WriteLine("Task 2");
+
+            Console.WriteLine("Task 3");
+            Console.WriteLine("Введите натуральное число");
+            var n = int.Parse(Console.ReadLine());
+            Console.WriteLine($"Квадраты натуральных чисел не превышающих {n}:");
+            int j = 1;
+            while (j * j < n)
+            {
+                Console.Write($"{j * j++} ");
+            }
+            Console.WriteLine();
+
+
+            Console.WriteLine("Task 4");
+            Console.WriteLine("Введите натуральное число");
+            var m = int.Parse(Console.ReadLine());
+            Console.WriteLine("Введите целое число от 0 до 8 включительно");
+            var k = int.Parse(Console.ReadLine());
+            int result = Task4(m, k);
+            Console.WriteLine($"Сумма цифр числа {m} не превышающих числа {k}: {result}");
+            Console.WriteLine();
+
+
+            Console.WriteLine("Task 5");
+            Console.WriteLine("Введите первое натуральное число");
+            var f = int.Parse(Console.ReadLine());
+            Console.WriteLine("Введите второе натуральное число");
+            var g = int.Parse(Console.ReadLine());
+            
+            for (int i = 0; i < (f * g); i++)
```

Comment:
Можно было начать меньшего из чисел
```suggestion
            for (int i = Math.Min(f, g); i < (f * g); i++)
```